api/coverage.proto

```diff
@@ -9,6 +9,9 @@ option java_outer_classname = "ApiCoverageProto";
 message OptionsData {
     string id = 1;
     repeated OptionsChart optionsChart = 2;
+    string company_id = 3;
```

Comment:
@Woody1193 We don't really expose company ids to external clients. It's only internal to us. We use `billingInternalId` in Blue and non-Blue APIs.

Reply:
@flowerinthenight I'll remove these then. Sorry, but I saw them on the RI/SP responses so I thought that was okay. Should I remove the `company_id` field from those as well then or do you want to leave them as tech debt?

Reply:
Not really remove but replace to `billingInternalId`.